packages/@monorepo-utils/package-utils/package.json

```
@@ -63,7 +63,8 @@
   "dependencies": {
     "globby": "^11.0.1",
     "load-json-file": "^6.2.0",
-    "upath": "^2.0.1"
+    "upath": "^2.0.1",
+    "yaml": "2.1.3"
```

use ^ instead of pin version

```suggestion
    "yaml": "^2.1.3"
```

---

I apply this patch and merge it.